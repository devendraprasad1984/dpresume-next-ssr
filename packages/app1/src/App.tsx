import React, {useState} from 'react';
import './App.css';
import {Button} from "@dp/app2";

function App() {
    const [count, setCount] = useState(0)
    return (
        <div className="App">
            <h2>count: {count}</h2>
            <Button text='click me' onClick={() => setCount(p => ++p)}/>
        </div>
    );
}

export default App;
