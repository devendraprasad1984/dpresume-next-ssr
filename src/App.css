span, div, p {
    font-family: inherit;
    padding-left: 3px;
    padding-right: 3px;
}

span.link:has(a.active) {
    border-bottom: 2px solid var(--gray-color);
}

.active {
    background: var(--gradient-image-color);
    /*background-image: var(--gradient-image-color);*/
    -webkit-background-clip: text;
    -webkit-text-fill-color: transparent;
    font-size: 25px;
}

.nav {
    background-color: var(--black-color);
}

.nav ul {
    list-style-type: none;
    margin: 0;
    padding: 0;
    display: flex;
    flex-direction: row;
    justify-content: flex-start;
    align-items: center;
    align-content: center;
}

.nav ul li,
.nav ul li a {
    padding: 5px;
    margin: 2px;
    color: var(--grad-color2) !important;
}

h1,
h2,
h3,
h4,
h5,
h6,
.tld {
    color: var(--black-color);
    font-weight: bolder;
    cursor: pointer;
    text-decoration: none;
    padding: 0;
    margin: 0;
}

hr {
    overflow: visible; /* For IE */
    padding: 0;
    border: none;
    border-top: medium double var(--medium-violet);
    color: var(--grad-color2);
    text-align: center;
}

#idLinks a {
    font-size: var(--size10px);
}

h1 {
    display: flex;
    vertical-align: middle;
}

.size10 {
    font-size: 12px !important;
}

.size15 {
    font-size: 18px !important;
}

.size20 {
    font-size: 20px !important;
}

.size25 {
    font-size: 25px !important;
    font-weight: bolder;
}

.size30 {
    font-size: 30px !important;
    font-weight: bolder;
}

div > button,
div > .btn {
    margin: 2px;
}

.size35 {
    font-size: 35px !important;
    font-weight: bolder;
}
.size45 {
    font-size: 45px !important;
    font-weight: bolder;
}
.pcenter {
    display: flex;
    justify-content: center;
    align-content: center;
    align-items: center;
}

.mcard span {
    width: 100%;
    height: 100%;
}

.center {
    display: flex;
    justify-content: center;
    align-items: center;
    text-align: center;
    word-break: break-all;
    align-content: center;
}

.parent-center{
    position: relative;
}
.child-center{
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
}

.notfound .notfound-404 {
    position: relative;
    height: 180px;
}

.margin-ud {
    margin-top: var(--size10px);
    margin-bottom: var(--size10px);
}

.margin-rl {
    margin-left: var(--size10px);
    margin-right: var(--size10px);
}

.padding-ud {
    padding-top: var(--size10px);
    padding-bottom: var(--size10px);
}

.padding-rl {
    padding-left: var(--size10px);
    padding-right: var(--size10px);
}

b {
    color: var(--purple-color) !important;
}

.pright {
    display: flex;
    flex-direction: row;
    width: 100%;
    /*height: 100vh;*/
}

/*.icons {*/
/*    font-family: 'Material Icons';*/
/*    font-weight: normal;*/
/*    font-style: normal;*/
/*    letter-spacing: normal;*/
/*    text-transform: none;*/
/*    display: inline-block;*/
/*    white-space: nowrap;*/
/*    word-wrap: normal;*/
/*    direction: ltr;*/
/*    -webkit-font-feature-settings: 'liga';*/
/*    -webkit-font-smoothing: antialiased;*/
/*}*/

/*.icons:hover, a:hover, i:hover, .fab:hover, .fas:hover, .far:hover {*/
/*    color: var(--gray-color);*/
/*}*/

.ml2 {
    font-weight: 900;
    /*font-size: 3.5em;*/
}

.ml2 .letter {
    display: inline-block;
    /*line-height: 1em;*/
}

.ml6 {
    position: relative;
    font-weight: 900;
}

.ml6 .text-wrapper {
    position: relative;
    display: inline-block;
    padding-top: 0.2em;
    padding-right: 0.05em;
    padding-bottom: 0.1em;
    overflow: hidden;
}

.ml6 .letter {
    display: inline-block;
}

.ml13 {
    text-transform: uppercase;
    letter-spacing: 0.1em;
    /*font-weight: 800;*/
}

.ml13 .letter {
    display: inline-block;
}

.cursel {
    background-color: #ceadf5;
    font-weight: bolder;
}

.capt {
    background-color: grey;
    width: 300px;
    height: 100px;
}

#mainCaptcha {
    font-size: 15pt;
}

iframe {
    border: 0;
    /*-moz-border-radius: 5px;*/
    /*-webkit-border-radius: 5px;*/
    /*border-radius: 5px;*/
    /*background-color: white;*/
    overflow: auto;
    word-wrap: break-word;
}

pre {
    line-height: 1.2;
    -ms-overflow-style: scrollbar;
    white-space: pre-wrap; /* css-3 */
    white-space: -moz-pre-wrap; /* Mozilla, since 1999 */
    white-space: -pre-wrap; /* Opera 4-6 */
    white-space: -o-pre-wrap; /* Opera 7 */
    word-wrap: break-word; /* Internet Explorer 5.5+ */
    overflow: auto;
    color: var(--black-color);
    font-size: 14px;
}

/*.preline *::before {*/
/*    content: "â€“";*/
/*    padding-right: 8px;*/
/*    font-size: 20px;*/
/*}*/

.page {
    position: relative;
    overflow-x: hidden;
    overflow-y: auto;
    height: 100%;
    width: 100%;
    background-color: white;
}

footer,
.bottom {
    padding: 3px;
    bottom: 0;
    left: 0;
    margin-top: 5em;
    /*position: fixed;*/
    width: 100%;
}

section {
    /*padding: 4px;*/
    left: 0;
    position: relative;
    width: 100%;
}

.load {
    z-index: 2;
    height: 30px;
    left: 40%;
    top: 5px;
    position: absolute;
    padding: 4px;
    display: none;
    opacity: 0.7;
}

.text-black {
    color: var(--black-color);
}

.column {
    flex: 50%;
    height: 300px;
}

.right-flex {
    display: flex;
    align-content: flex-end;
    align-items: flex-end;
    align-self: flex-end;
    text-align: right;
}

.right {
    /*float: right;*/
    display: flex;
    text-align: right;
    justify-content: flex-end;
    align-content: flex-end;
    align-items: flex-end;
    flex-wrap: wrap;
}

.left {
    float: left;
}

.height400 {
    min-height: 60vh;
}

.star {
    color: var(--purple-color);
    cursor: pointer;
}

.star:hover {
    color: var(--gray-color);
    float: right;
}

.noHover div > span {
    pointer-events: auto;
}

.content-left {
    padding-left: 5px;
    width: var(--left-panel-width);
}

.gapSwitchMobile {
    margin-top: 10rem;
}

.link {
    list-style: none;
    margin: 0;
}

.link a {
    padding: 5px;
    margin: 5px;
}

.content-right {
    width: var(--right-panel-width);
    padding-left: var(--size10px);
    padding-right: var(--size10px);
    float: left;
}

.top {
    top: 0;
    left: 0;
}

.name {
    font-weight: bold;
    font-size: 28px;
}

.name-lead {
    font-weight: bold;
    font-size: 20px;
}

p {
    margin-top: 0;
    margin-bottom: 1rem;
}

.pad10,
.pad10 > .btn {
    padding: var(--size10px);
}

.margin10 {
    padding: var(--size10px);
}

.btn,
button {
    display: inline-block;
    font-weight: bold;
    text-align: center;
    white-space: nowrap;
    vertical-align: middle;
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
    /*border: 1px solid transparent;*/
    padding: 10px 10px;
    /*font-size: 1rem;*/
    line-height: 1.2;
    border-radius: 0.25rem;
    cursor: pointer;
    /*color: var(--white) !important;*/
    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out,
    border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
}

code,
kbd,
samp {
    font-family: SFMono-Regular, Menlo, Monaco, Consolas, "Liberation Mono",
    "Courier New", monospace;
    font-size: 1em;
}

.plzwait {
    font-size: 12px;
}

.flexbox {
    grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));
    display: grid;
}

.flexbox-video {
    grid-template-columns: repeat(auto-fill, minmax(300px, 1fr));
    display: grid;
}

.cards > span,
.cards > a {
    font-size: 12px;
    padding: 5px;
    margin: 3px;
    height: 100px;
    vertical-align: middle;
    text-align: center;
    box-sizing: border-box;
    border-radius: 5px;
    box-shadow: 0 4px 4px 0 var(--gray-color);
    opacity: 0.7;
}

.cards span:hover,
.cards a:hover {
    background-image: var(--gradient-image-color);
    color: var(--black-color) !important;
}

.video-cards > span,
.video-cards > a {
    padding: var(--size10px);
    margin: var(--size10px);
    height: 350px;
    vertical-align: middle;
    text-align: center;
    box-sizing: border-box;
    border-radius: 5px;
    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.2);
    opacity: 0.7;
}

.mcard {
    display: inline-block;
    vertical-align: middle;
    line-height: normal;
    letter-spacing: 1px;
}

.row {
    display: flex;
    justify-content: space-between;
    flex-direction: row;
}

.row * {
    margin: 2px;
}

.flex1 {
    flex: 1;
    flex-basis: auto;
}

.col {
    display: flex;
    /*justify-content: space-between;*/
    flex-direction: column;
}

.bottom a {
    font-size: var(--size10px);
    color: var(--gray-color);
}

a {
    cursor: pointer;
    font-weight: bold;
    text-decoration: none;
    background-color: transparent;
}

/*.droplet {*/
/*    background: var(--grad-color1);*/
/*    border-radius: 5% 5%;*/
/*    transition: all 1s;*/
/*    transition-timing: ease-in-out;*/
/*}*/

.overlay {
    height: 100%;
    width: 0;
    position: fixed;
    z-index: 1000;
    top: 0;
    left: 0;
    background-color: var(--grad-color2);
    overflow-x: hidden;
    display: none;
    opacity: 0.85;
    /*transition: 0.5s;*/
}

.overlay-content {
    color: white;
    position: relative;
    /*top: 1%;*/
    width: calc(100% - 150px);
    padding: 30px;
    margin-top: 5px;
    overflow-x: hidden;
    overflow-y: auto;
}

.click,
i,
.icons {
    cursor: pointer;
}

.labelx {
    font-weight: bold;
}

.time {
    font-size: var(--size10px);
}

.overlay .closebtn {
    position: fixed;
    right: 45px;
    float: right;
    margin-right: 50px;
}

.box {
    border: 1.5px solid var(--gray-color);
    border-radius: 0.3rem;
    padding: 5px;
    left: 0;
    margin-bottom: var(--size10px);
    line-height: 1.5rem;
    box-shadow: 2px 2px 10px var(--gray-color);
}

.bl {
    font-weight: bolder;
}

.primary {
    background-color: #a4a0a0 !important;
    color: var(--white) !important;
}

.secondary {
    background-color: #4b5054 !important;
    color: var(--white) !important;
}

.success {
    background-color: #28a745 !important;
    color: var(--white) !important;
}

.info {
    background-color: #7055c6 !important;
    color: var(--white) !important;
}

.warning {
    background-color: #ffc107 !important;
    color: var(--white) !important;
}

.wid60 {
    width: 60%;
}

.white {
    background-color: var(--white) !important;
}

.xprimary {
    color: #3881d0 !important;
}

.xwhite {
    color: var(--white) !important;
}

.xsecondary {
    color: #6c757d !important;
}

.xgray {
    color: var(--gray-color) !important;
}

.xsuccess {
    color: #28a745 !important;
}

.xinfo {
    color: #3dadcd !important;
}

.xwarning {
    color: #ffc107 !important;
}

.xred {
    color: red !important;
}

.xwhite {
    color: white !important;
}

.xblue {
    color: #3e9fe9 !important;
}

.xviolet {
    color: var(--medium-violet) !important;
}

.xorange {
    color: #ea6135 !important;
}

.danger {
    background-color: #ee5664 !important;
    color: var(--white) !important;
}

.white {
    background-color: white !important;
}

.light {
    background-color: #f8f9fa !important;
}

.green {
    background-color: #49ec49 !important;
}
.purple {
    background-color: #8549ec !important;
}
.dark {
    background-color: #343a40 !important;
}

.dp {
    height: 100%;
    width: 100%;
    object-fit: cover;
    overflow: hidden;
}

.portrait {
    height: 100%;
    width: 40px;
}

.landscape {
    height: 100%;
    width: 80px;
}

.square {
    height: 100%;
    width: 100px;
}

#barStatus {
    position: absolute;
    width: 1%;
    height: 20px;
    display: none;
}

.badge {
    display: inline-block;
    padding: 0.25em 0.4em;
    line-height: 1.4;
    /*font-size: 11px;*/
    text-align: center;
    white-space: nowrap;
    vertical-align: baseline;
    border-radius: 0.25rem;
}

.imgPic,
#imgPic {
    width: 200px;
    height: 200px;
}

.runbar {
    border-bottom: 3px solid var(--black-color);
}

.img {
    width: 100%;
    height: 250px;
    padding: 5px;
    display: inline-block;
    vertical-align: middle;
}

.img-animate {
    border-radius: 100%;
    -webkit-animation: border-transform-default 10s linear infinite alternate forwards;
    animation: border-transform-default 10s linear infinite alternate forwards;
}

.imgDonate {
    width: 300px;
    height: 250px;
    padding: 5px;
}

.imgIcon {
    display: inline-flex;
}

.openFile {
    position: relative;
    overflow: hidden;
    margin-bottom: 2px;
}

.openFile input.open {
    position: absolute;
    top: 0;
    right: 0;
    margin: 0;
    padding: 0;
    font-size: 20px;
    cursor: pointer;
    opacity: 0;
    filter: alpha(opacity=0);
}

#jsEditor,
#pythonEditor,
#sqlEditor {
    font-weight: bolder;
    position: relative;
    width: 100%;
    height: 500px;
    font-size: 12px;
    overflow: auto;
}

#codeBlock,
.codeBlock {
    padding: 3px;
    overflow: auto;
    font-size: var(--size10px);
    color: var(--gray-color);
    width: 100%;
    height: 100%;
}

#codeBlock::-webkit-scrollbar {
    width: 0;
}

@keyframes lineanimate {
    0% {
        left: 0;
    }

    50% {
        left: 100%;
    }

    0% {
        left: 0;
    }
}

@keyframes border-transform-default {
    0%,
    100% {
        border-radius: 63% 37% 54% 46%/55% 48% 52% 45%;
    }
    14% {
        border-radius: 40% 60% 54% 46%/49% 60% 40% 51%;
    }
    28% {
        border-radius: 54% 46% 38% 62%/49% 70% 30% 51%;
    }
    42% {
        border-radius: 61% 39% 55% 45%/61% 38% 62% 39%;
    }
    56% {
        border-radius: 61% 39% 67% 33%/70% 50% 50% 30%;
    }
    70% {
        border-radius: 50% 50% 34% 66%/56% 68% 32% 44%;
    }
    84% {
        border-radius: 46% 54% 50% 50%/35% 61% 39% 65%;
    }
}

/*Scratch Coupon Card*/
.code-content {
    padding: 20px;
    background-color: #e4e1e1;
    height: 300px;
}

.code-content h2 {
    text-transform: uppercase;
    text-align: center;
    font-size: 20px;
    -webkit-user-select: none; /* Chrome all / Safari all */
    -moz-user-select: none; /* Firefox all */
    -ms-user-select: none; /* IE 10+ */
    user-select: none; /* Likely future */
}

.code-content .code-area img {
    -webkit-user-select: none; /* Chrome all / Safari all */
    -moz-user-select: none; /* Firefox all */
    -ms-user-select: none; /* IE 10+ */
    user-select: none; /* Likely future */
}

.code-content .txt-code {
    width: 200px;
    margin: 0 auto;
    text-align: center;
    padding: var(--size10px);
    border: 3px dotted #9a9696;
    position: relative;
    margin-top: var(--size10px);
}

.code-content .code-area {
    text-align: center;
}

.code-content img.sc-copy {
    width: 20px;
    position: absolute;
    right: var(--size10px);
    top: var(--size10px);
}

.code-content .tooltip {
    display: inline-block;
}

.code-content .tooltip .tooltiptext {
    visibility: hidden;
    width: 160px;
    background-color: #555;
    color: #fff;
    text-align: center;
    border-radius: 6px;
    padding: 5px;
    position: absolute;
    z-index: 1;
    bottom: 150%;
    left: 50%;
    margin-left: -75px;
    opacity: 0;
    transition: opacity 0.3s;
}

.code-content .tooltip .tooltiptext::after {
    content: "";
    position: absolute;
    top: 100%;
    left: 50%;
    margin-left: -5px;
    border-width: 5px;
    border-style: solid;
    border-color: #555 transparent transparent transparent;
}

.code-content .tooltip:hover .tooltiptext {
    visibility: visible;
    opacity: 1;
}

/*jquery.modal*/
.cyno-overlay {
    background: #333;
    opacity: 0.8;
    z-index: 1000;
    width: 100%;
    height: 100%;
    position: fixed;
    top: 0;
    left: 0;
}

.cyno-modal {
    background: #fff;
    z-index: 1001;
    box-shadow: 3px 3px 20px #000;
}

.cyno-closeBtn {
    font: 16px sans-serif;
    padding: 4px 0px;
    color: #fff;
    text-align: center;
    width: 30px;
    height: 30px;
    border-radius: 10px;
    background: #000;
    border: 3px solid #fff;
    box-shadow: -3px 3px 5px #000;
    display: block;
    position: absolute;
    top: -15px;
    right: -15px;
    z-index: 1002;
}

.input-wrapper {
    position: relative;
    margin: 20px auto;
}

.form-control {
    display: block;
    line-height: 1.4em;
    margin: 0;
    padding-left: 10px;
    width: 100%;
    font-size: medium;
    color: var(--purple-color);
    border: 1px solid var(--black-color);
    background-color: var(--white);
    border-radius: 5px;
    -webkit-box-sizing: border-box;
    -moz-box-sizing: border-box;
    box-sizing: border-box;
    font-weight: 500;
}

.form-control:focus {
    border: 1px solid var(--black-color);
}

.control-label {
    display: block;
    position: absolute;
    opacity: 0;
    bottom: 1.9rem;
    font-weight: bold;
    color: var(--purple-color);
    transition: 0.3s ease-in-out transform;
    font-size: 13px;
}

.form-control:placeholder-shown + .control-label {
    visibility: hidden;
    z-index: -1;
    transition: 0.2s ease-in-out;
}

.form-control:not(:placeholder-shown) + .control-label,
.form-control:focus:not(:placeholder-shown) + .control-label {
    visibility: visible;
    z-index: 1;
    opacity: 1;
    transform: translateY(-10px);
    transition: 0.2s ease-in-out transform;
}

.show {
    display: block;
}

.hide {
    display: none;
}

.modal-wrapper {
    width: 100vw;
    height: 100vh;
    min-height: 100vh;
    min-width: 100vw;
    position: fixed;
    top: 0;
    left: 0;
    background-color: rgba(0, 0, 0, 0.7);
    transition: 0.4s;
    /*visibility: hidden;*/
    /*opacity: 0.9;*/
    z-index: 12002;
}

.modal-content {
    padding: 10px;
    max-width: 100vw;
    width: 90vw;
    position: relative;
    top: 10%;
    left: 0;
    right: 0;
    margin: 0 auto;
    z-index: 1001;
    background-color: var(--white);
    border-radius: 5px;
    color: var(--gray-color);
    height: calc(90vh - 10%);
    overflow: auto;
}

.modal-opened {
    opacity: 1;
    visibility: visible;
}
