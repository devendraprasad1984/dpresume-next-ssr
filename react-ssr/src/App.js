import logo from './logo.svg';
import './App.css';

function App() {
  return (
      <div>
        dpresume.com
      </div>
  );
}

export default App;
