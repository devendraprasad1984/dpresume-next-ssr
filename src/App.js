import React, { useState } from "react";
import "./App.css";
import "./console.css";
import Main from "./components/main";
import Welcome from "./components/screens/welcome";
import { config } from "./configs/config";

const nameFromLocalStorage = localStorage.getItem(
  config.enums.localStorage.name
);
const isNameSet =
  nameFromLocalStorage !== "" &&
  nameFromLocalStorage !== null &&
  nameFromLocalStorage !== undefined;
// console.log('nameFromLocalStorage',nameFromLocalStorage, isNameSet)

function App(props) {
  const [canWeShowWelcome, setCanWeShowWelcome] = useState(!isNameSet);
  const handleCloseWelcome = (name) => {
    // console.log('closing in main app. found', name)
    localStorage.setItem(config.enums.localStorage.name, name);
    setCanWeShowWelcome(false);
  };
  return canWeShowWelcome ? <Welcome onClose={handleCloseWelcome} /> : <Main />;
  // return <Main />;
}

export default App;
