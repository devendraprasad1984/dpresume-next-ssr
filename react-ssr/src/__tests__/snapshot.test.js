import React from "react";

import {shallow} from 'enzyme'
import App from "../App";
import toJson from "enzyme-to-json";
import Home from "../components/screens/home";
import Education from "../components/screens/education";

//checks for the unexpected changes in component tree
//one of the importatn checks
describe('snapshot changes of components', () => {
    it('app snapshots', () => {
        const tree = shallow(<App/>)
        expect(toJson(tree)).toMatchSnapshot()
    })
    it('home snapshots', () => {
        const home = shallow(<Home/>)
        expect(toJson(home)).toMatchSnapshot()
    })
    it('eduction snapshots', () => {
        const eduction = shallow(<Education/>)
        expect(toJson(eduction)).toMatchSnapshot()
    })
})
