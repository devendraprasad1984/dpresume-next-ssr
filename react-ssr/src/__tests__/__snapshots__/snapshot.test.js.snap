// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot changes of components app snapshots 1`] = `<Main />`;

exports[`snapshot changes of components eduction snapshots 1`] = `
<div>
  <OneLinerHeader />
  <BasicDisplay
    list={
      Array [
        "<b>PGDITM</b>, SCDL, Symbiosis, Pune, India (2017)",
        "<b>BTech IT</b>, MDU Rohtak, Haryana, India (2007)",
      ]
    }
  />
</div>
`;

exports[`snapshot changes of components home snapshots 1`] = `
<div>
  <OneLinerHeader />
  <BasicDisplay
    list={
      Array [
        "I have 14+ years wide experience in software development industry.",
        "I work(ed) on primarily <b>React.Js, Js, Html, Python3, django3, SQL, Postgres, git</b> & secondarily <b>NodeJs, Tornado, PlSql, mysql, php7, Vue, Svelte</b>",
        "I am versatile & passionate developer with decent soft skills.",
        "I am lead developer over Expansion packs for finance reporting, working on python, django, postgres, React.JS and REST APIs",
        "I design secure api calls through HS256 JWT, Crypto Signer, CSRF, cookies, white-listing and short lived tokens",
        "I do Code . Collaborate . Build . Change . Incident . Release . Triage",
        "I help in recruiting, grow team and work on enabling new team mates",
        "I am a code reviewer,mentor and face to dev team",
        "I use JIRA & Leankit for agility, bug tracking and backlog grooming",
        "I know SDLC, Incident | Change | Release management",
      ]
    }
  />
  <HomeDemo />
</div>
`;
